source/extensions/filters/http/grpc_stats/grpc_stats_filter.h

```diff
@@ -24,6 +24,12 @@ struct GrpcStatsObject : public StreamInfo::FilterState::Object {
     return msg;
   }
 
+  ProtobufWkt::Value serializeAsProtoValue() const override {
+    ProtobufWkt::Value val;
+    val.set_string_value(absl::StrCat(request_message_count, ",", response_message_count));
```

Comment:
This is not quite right. You actually need to recreate the `envoy::extensions::filters::http::grpc_stats::v3::FilterObject` as a  Value proto. It is not hard since it just has two uint64 fields. The code would look something like this:
```
::ProtobufWkt::Value value;
auto *struct_value = value.mutable_struct_value();
::ProtobufWkt::Value::Value string_value;
string_value.set_string_value(absl::StrCat(request_message_count));
(*struct_value->mutable_fields())["request_message_count"] = string_value;

// same for the response_message_count
```